independent-projects/bootstrap/runner/src/main/java/io/quarkus/bootstrap/logging/QuarkusDelayedHandler.java

```diff
@@ -285,6 +290,41 @@ private synchronized void activate() {
                 publishToNestedHandlers(record);
             }
         }
+
+        for (CategoryAndLevel entry : droppedRecords) {
+            if (Logger.getLogger(entry.category).isLoggable(entry.level)) {
+                reportError(
+                        "The delayed handler's queue was overrun and log record(s) were lost. Did you forget to configure logging?",
```

Comment:
I think that it would make sense to include the logger names (and levels) in the message, i.e. something like:
```
The delayed handler's queue was overrun and log record(s) were lost:
   - org.acme.Foo: DEBUG, TRACE
   - com.acme.Baz: DEBUG 
````

Reply:
Message improved in [`14aa776` (#21473)](https://github.com/quarkusio/quarkus/pull/21473/commits/14aa776bc38ebee286be649a9734e1b16e45c07a)